data/src/main/java/com/example/data/CustomerRepository.java

```diff
@@ -0,0 +1,12 @@
+package com.example.data;
+
+import java.util.List;
+
+import org.springframework.data.repository.CrudRepository;
+
+public interface CustomerRepository extends CrudRepository<Customer, Long> {
+
+    List<Customer> findByLastName(String lastName);
+
+    Customer findById(long id);
```

Comment:
I believe it is redundant. It is already a part of  CrudRepository and you do not need to redefine it if you do not want to change it somehow

Reply:
Question: what happens if there is no object with such Id?